js/iiif-iabookreader_strawberry.js

```diff
@@ -40,10 +40,138 @@
                             maxWidth: 800,
                             imagesBaseURL: 'https://cdn.jsdelivr.net/gh/internetarchive/bookreader@4.2.0/BookReader/images/',
                         };
-                        console.log('initializing iabookreader')
+                        console.log('initializing IABookreader')
                         var br = new BookReader(options);
                         br.init();
                     }
 
                 })}}
 })(jQuery, Drupal, drupalSettings);
+
+// override setupTooltips()
+// to enable system tooltip
+BookReader.prototype.setupTooltips = function() {
```

Comment:
This is awesome. I will merge this to have this awesome functionality in our book reader. Next step could be to move all Prototype functions maybe into https://github.com/esmero/format_strawberryfield/blob/master/js/plugin.iiif-iabookreader_strawberry.js, what do you think @giancarlobi ? Would make the main drupal js cleaner?